### PR TITLE
feat: 🎸 make background color independent of platform

### PR DESCRIPTION
**Pull request by theashraf**
## Type of change

- [ ]  Patch: Bug (non-breaking change which fixes an issue)
- [x]  Minor: New feature (non-breaking change which adds functionality)
- [ ]  Major: Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] This is something we need to do.

### CODE REVIEW

File: packages/web/cpp/renderer.cpp [hunk 1]
```diff
@@ -62,6 +62,18 @@ class __attribute__((visibility("default"))) Renderer
 
         resize(width, height);
 
+        // create background shape
+        background = Shape::gen();
```
Comment (hermet) [inline packages/web/cpp/renderer.cpp:66]: auto background = Shape::gen();
this->background = background.get();

**Review by hermet — COMMENTED** [review]
Please check a comment.
It needs to retain a pointer from the unique pointer.

File: packages/web/cpp/renderer.cpp [hunk 2]
```diff
@@ -204,9 +240,11 @@ class __attribute__((visibility("default"))) Renderer
     string errorMsg;
     unique_ptr<SwCanvas> canvas = nullptr;
     unique_ptr<Animation> animation = nullptr;
+    unique_ptr<Shape> background = nullptr;
```
Comment (hermet) [inline packages/web/cpp/renderer.cpp:243]: Shape* background = nullptr;

File: packages/web/cpp/renderer.cpp [hunk 3]
```diff
@@ -177,6 +208,14 @@ class __attribute__((visibility("default"))) Renderer
         animation->picture()->scale(scale);
         animation->picture()->translate(shiftX, shiftY);
 
+        if (background)
+        {
+
+            background->reset();
+            background->appendRect(shiftX, shiftY, psize[0], psize[1]);
+            background->scale(scale);
```
Comment (theashraf) [inline packages/web/cpp/renderer.cpp:216]: @hermet when the background is not scaled like the picture, it doesn't align properly on the canvas, as shown in the screenshot. Is this acceptable, or is there a more effective solution?

image (1)

**Review by afsalz — APPROVED** [review]
Looks good 🚀

**Pull request merged**